pyproject.toml

```
@@ -0,0 +1,78 @@
+[project]
+dynamic = ["version", "dependencies"]
+name = "kinto-emailer"
+description = "Kinto emailer plugin"
+readme = {file = "README.rst", content-type = "text/x-rst"}
```

According to [this example](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#configuring-setuptools-using-pyproject-toml-files), it seems that we could do:

```suggestion
readme = "README.rst"
```